notebooks/official/feature_store/gapic-feature-store.ipynb

```diff
@@ -171,7 +171,7 @@
       },
```

Comment:
<p>Is there a particular reason we want to specify the version? If not we can simply use</p><p><code>! pip install {USER_FLAG} --upgrade google-cloud-aiplatform</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Agreed, we try to avoid pinning the version. We run regular CI checks which will tell us if there's a breaking change.